sopel.cfg-dist

```diff
@@ -0,0 +1,43 @@
+[core]
+## Bot presence and identity
+nick = mechasqueak-test
+# channels = #rattest,#botabuse
+host = irc.mibbit.com
+use_ssl = true
+verify_ssl = False
+port = 6697
+# auth_method = sasl
+# auth_password = topsecret
+# auth_username = mechasqueak-test
+
+prefix = \!
+help_prefix = !
+
+## Admins
+# owner = OwnerNameHere
+# admins = Joe,Bob
+
+## Where to find modules
+extra = sopel-modules
+
+## For production/testing
+enable = admin,help,rat-board,rat-facts,rat-search
```

Comment:
Reload should be in production. It's limited to bot.admin/bot.owner only. A reload is faster than an entire bot restart and doesn't interfere with other modules.